src/platform/forms-system/src/js/review/submit-states/ErrorLinks.jsx

```diff
@@ -69,21 +69,19 @@ const ErrorLinks = props => {
                 {errors.map(error => (
                   <li key={error.name}>
                     {error.chapterKey ? (
-                      <a
+                      <va-button
                         href="#"
```

Comment:
I think these links need to remain as links. If you look at the [reviewErrors docs](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/docs/reviewErrors.md) you'll see a screenshot of what this renders. Having these all as primary buttons in this list with a lot of text would be overwhelming for users.

Also, are we sure scrolling to a just-opened accordion works?

Reply:
There is no utility class for styling buttons as links any more. Having these as links does break the material honesty convention, but I think having buttons with a lot of text inside is worse. We can bring this up with an a11y expert.

Reply:
@laflannery @jeana-adhoc 

Do either of you have a position on this? Both solutions here are easy, it's just a matter of which tradeoffs we want to take.

Reply:
@zack and @Mottie I'm just trying to catch up - we are looking at [this error message](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/docs/images/error-text-based-on-error-name.png) correct? Where items such as "Has training pay" and "POW status" are interactive items that do something? I thought these were anchor links that take the user further down the page to the correct section

Reply:
Thanks @Mottie. I think links would make sense here rather than buttons. I agree with your statement that having this many buttons probably is not going to be great. Also, thinking about this in terms of what is happening - when I activate these items - they are going to take me somewhere, navigate me to the particular section of the page/form where I need to go to edit and that navigational element is what a link is for rather than a button. So that's my reasoning behind why I think links would make sense here.

Reply:
The more I try to generate these on the page the less convinced I am they even work? @Mottie maybe you can help me see what I'm missing here:

If we search for `ErrorLinks` in the codebase we get these results:

```
👾  vets-website(main) $ rg ErrorLinks
src/platform/forms-system/src/js/review/submit-states/ErrorLinks.jsx
8:const ErrorLinks = props => {
110:ErrorLinks.propTypes = {
121:)(ErrorLinks);
123:export { ErrorLinks };

src/platform/forms-system/src/js/review/submit-states/ValidationError.jsx
9:import ErrorLinks from './ErrorLinks';
23:    <ErrorLinks appType={appType} testId={testId} />

src/platform/forms-system/test/js/review/submit-states/ErrorLinks.unit.spec.jsx
7:import { ErrorLinks } from '../../../../src/js/review/submit-states/ErrorLinks';
9:describe('<ErrorLinks />', () => {
18:    const view = render(<ErrorLinks {...props} />);
42:    const view = render(<ErrorLinks {...props} />);
65:    const view = render(<ErrorLinks {...props} />);
87:    const view = render(<ErrorLinks {...props} />);
91:    view.rerender(<ErrorLinks {...props} errors={[]} />);
```

Which is one set of tests, one invocation, and one definition. Which is to say they're used in exactly one place.

In `ErrorLinks.jsx` the `errors` array that is used to generate the links in question comes from `props`, but in the _only_ place `ErrorLinks` is ever used (ValidationError.jsx#23), no errors are passed in, which means we'd see something like this:

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/2482403/81ec6ef8-539a-4dfc-be42-f70136c7cc6a)

I think the invocation on `ValidationError.jsx#23` needs to be passing `props.formErrors` into `ErrorLinks` using its `errors` prop. But with a regression like this I think it should also get tests. And maybe in that work we can come back to the styling of the buttons/links? It feels like some serious scope creep on this ticket which is supposed to be fixing a11y issues on the accordion and edit buttons.

Reply:
Deleted my last message because I'm still trying to remember that we're using redux and `mapStateToProps`. 😨

Reply:
Links are back :)